test/system/505-networking-pasta.bats

```diff
@@ -252,6 +252,10 @@ function pasta_test_do() {
     # Get server output, --follow is used to wait for the container to exit,
     run_podman logs --follow $cname
     # which should give us the expected output back.
+    # ...except, sigh, #23482: seems to be a bug in socat, issues spurious warning
+    if [[ "$recv" =~ md5sum ]]; then
```

Comment:
nit: should we match `EXEC`? while unlikely the md5sum command might be changed to a different one or a new command is added,etc.. causing the error to happen again...

Reply:
That's not a nit, it's an outright logic bug. Thanks for catching it. Fixed & pushed & CI is green.